### ISSUE TITLE
Enable greenkeeper

### ISSUE THREAD

**Issue by cemremengu** [fastify/fastify-routes#5]

**Comment by mcollina** [comment]
I don't know how to. https://account.greenkeeper.io/account/fastify#repositories does not show this repository.

**Comment by cemremengu** [comment]
Even after refreshing the repo list?

**Comment by mcollina** [comment]
You might want to create a new repo and push the code there. It might be due to moving the repo.

**Comment by cemremengu** [comment]
https://github.com/fastify/fastify-route

Can you rename or delete other repo ?

**Comment by cemremengu** [comment]
Yes this one. I created `fastify-route`. Once this repo is gone, I can use its name
